docs/source/records_mover.records.rst

```diff
@@ -0,0 +1,82 @@
+records\_mover.records package
+==============================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   records_mover.records.sources
+   records_mover.records.targets
+
+Submodules
+----------
+
+records\_mover.records.base\_records\_format module
+---------------------------------------------------
+
+.. autoclass:: records_mover.records.base_records_format.BaseRecordsFormat
+   :members:
+   :show-inheritance:
+   :special-members: __init__
```

Comment:
Most of the time I chose to bring up key public API classes into the `__all__` of `records_mover` or `record_mover.records`, which also ensures that Sphinx will document it there.  I then left out the implementation module where it was defined from Sphinx so it didn't get double-documented, and so that we can move things around without folks doing deep 'import' statements.

This is a case where I couldn't get away with that, as Sphinx documented it as a base class of DelimitedRecordsFormat and ParquetRecordsFormat--leaving this out broke the link.  I couldn't figure how to convince it to do otherwise.